lib/ecto/postgres/pg_rational_migrations.ex

```diff
@@ -0,0 +1,209 @@
+use Vtc.Ecto.Postgres.Utils
+
+defpgmodule Vtc.Ecto.Postgres.PgRational.Migrations do
+  @moduledoc """
+  Migrations for adding rational types, functions and constraints to a
+  Postgres database.
+  """
+  alias Vtc.Ecto.Postgres
```

Comment:
haven't tried this, but could we just `use Ecto.Migration` in this module, and then define the functions normally, without them being macros? doesn't look like the macros are really required, at the surface-level, but may be missing something as i read through this.

Reply:
That was a great idea! In: 25b5c7c81ef106fd54767d8f0e146f747c964dd9